include/bio_ik/goal.h

```diff
@@ -97,8 +97,8 @@ class GoalContext
 class Goal
 {
 protected:
-    bool secondary_;
     double weight_;
+    bool secondary_;
```

Comment:
Did you rearrange these data members to influence the size of the class? Presumably with the double before the bool it's packed more tightly than before?